examples/vision/probing_vits.py

```diff
@@ -0,0 +1,679 @@
+"""
+Title: Under the Hood of Vision Transformers
+Authors: [Aritra Roy Gosthipaty](https://twitter.com/ariG23498), [Sayak Paul](https://twitter.com/RisingSayak) (equal contribution)
+Date created: 2022/04/12
+Last modified: 2022/04/15
+Description: Probing into various variants of Vision Transformers.
```

Comment:
Avoid "various variants". Let's find a subtitle that adds details compared to the title, e.g. "Looking into the representations learned by different Vision Transformers variants."